### PR TITLE
Expose Onyx in web development

### PR DESCRIPTION
**Pull request by iwiznia**
### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/220006

### Tests
- Opened the app on web and checked that Onyx was available in the console. For staging testing this will not work, so test is the opposite: open console in web and check that typing Onyx in the console returns nothing (or undefined)
- Opened the app in ios and checked no errors

### CODE REVIEW

**Review by puneetlath — DISMISSED** [review]

**Comment by puneetlath** [comment]
Nice! This is great. Do you think we should add this to the documentation too so that people know about it? Maybe in the [debugging section](https://github.com/Expensify/App#debugging)?

**Comment by iwiznia** [comment]
Added it to the readme as you suggested

File: README.md [hunk 1]
```diff
@@ -121,6 +121,9 @@ Our React Native Android app now uses the `Hermes` JS engine which requires your
 3. You should now see a "Hermes React Native" target with an "inspect" link which can be used to bring up a debugger. If you don't see the "inspect" link, make sure the Metro server is running.
 4. You can now use the Chrome debug tools. See [React Native Debugging Hermes](https://reactnative.dev/docs/hermes#debugging-hermes-using-google-chromes-devtools)
 
+## Web
+
+To make it easier to test things in web, we expose the Onyx object to the window, so you can easily do `Onyx.set('bla', 1)`.
```
Comment (puneetlath) [inline README.md:126]:
```suggestion
To make it easier to test things in web, we expose the Onyx object to the window, so you can easily do `Onyx.set('bla', 1)`.

```

Seems to come through as h2 for some reason without the space after.

**Review by puneetlath — CHANGES_REQUESTED** [review]

**Comment by iwiznia** [comment]
Accepted the suggestion

**Review by puneetlath — APPROVED** [review]

**Pull request merged**